src/main/java/com/karankumar/bookproject/backend/statistics/RatingStatistics.java

```diff
@@ -41,7 +41,7 @@ public RatingStatistics(PredefinedShelfService predefinedShelfService) {
      */
     public Book findMostLikedBook() {
         if (readBooksRated.isEmpty()) {
-            return null;
+            return Optional.ofNullable(null);
```

Comment:
We should change the return of this method to Optional and then return empty optional